tests/Operation/WatchFunctionalTest.php

```diff
@@ -1370,11 +1371,11 @@ public function testOriginalReadPreferenceIsPreservedOnResume(): void
         }
 
         $changeStream = $operation->execute($secondary);
-        $previousCursorId = $changeStream->getCursorId(true);
-        $this->forceChangeStreamResume();
+        $previousCursorId = $changeStream->getCursorId();
+        $this->forceChangeStreamResume($secondary);
```

Comment:
Note: this method previously created the fail point on the primary server, instead of on the secondary server where the changestream was created.  This meant that the change stream was never resumed and the cursor ID was the same as before. We just didn't catch this mistake as the `assertNotSame` comparison always saw the objects as different as they didn't have the same identity. Lessons learned...

Reply:
It should be fixed on the 1.x branch, shouldn't it?

Reply:
Extracted to #1434. Will rebase when that PR is merged.